### ISSUE TITLE
False waring about Spring bean afterPropertiesSet method 

### ISSUE THREAD

**Issue by jadetang** [spotbugs/sonar-findbugs#196]
## Issue Description

False waring about Spring bean afterPropertiesSet method 

The sonarqube keeps giving me this message
`Correctness - Method of Singleton class writes to a field in an unsynchronized manner` 
on the Spring bean afterPropertiesSet method.
As I understood, this method will be called only once during the Spring bean’s lifecycle, just wanna know when sonarqube produces this message is it aware of the code actually runs under Spring framework?

## Environment

| Component          | Version |
| ------------------ | ------- |
| SonarQube          | ?????   |
| Sonar-FindBugs     | ?????   |
| Maven              | ?????   |
| Gradle             | ?????   |
| Java               | 8  |

## Code (If needed)

```java
public class Sample implement InitializingBean {
   private int value;

     @Override
  public void afterPropertiesSet()  {
    value = 1;  // give me warning here.
  }
}
```

**Comment by h3xstream** [comment]
This issues is raised by spotbugs.
Because it is not specific to the Sonar integration you can report it here :  https://github.com/spotbugs/spotbugs/issues

**Comment by Rahali** [comment]
hello,
any news about this issue ?
tnks.

**Comment by it-yayu** [comment]
How to resolve this warning

**Comment by gtoison** [comment]
Hello,
The sonarqube plugin only converts spotbugs issues into sonarqube issues. All the code analysis is performed by spotbugs so if you believe that this is a false positive you should report it here: https://github.com/spotbugs/spotbugs/issues
The code in the initial message does write to a field in an unsychronized manner, so maybe consider synchronizing the method or mark the issue as a false positive in sonarqube to simply ignore it
